conf/eslint-all.js

```diff
@@ -16,7 +16,11 @@ var fs = require("fs"),
 // Helpers
 //------------------------------------------------------------------------------
 
-var ruleFiles = fs.readdirSync(path.resolve(__dirname, "../lib/rules"));
+var ruleFiles = fs.readdirSync(
+        path.resolve(__dirname, "../lib/rules")
+    ).filter(function(file) {
```

Comment:
Do we need both filter and reduce? Could we just replace them both with a single forEach loop instead?